scripts/versions.sh

```diff
@@ -2,9 +2,28 @@
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
-# Set up the versions to be used
-AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.3'}
-SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.8'}
+set -e
+set -o pipefail
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+
+function setGO_VERSION() {
+    export GO_VERSION=$(getGO_VERSION)
+}
+
+function getGO_VERSION() {
+    echo $(getDepVersion go).$(grep -m1 GO_PATCH_VERSION $TELEPORTER_PATH/go.mod | cut -d ' ' -f3)
```

Comment:
instead of get/set functions for `GO_VERSION`, would we be able to just set the value with above `echo $(getDepVersion go).$(grep -m1 GO_PATCH_VERSION $TELEPORTER_PATH/go.mod | cut -d ' ' -f3)` then just `source scripts/versions.sh`?

Reply:
I think we *could* do this, but we would have to export it because the docker containers need it exported. I would rather anyone sourcing this file have finer-grained control over what they're exporting and what they're not.